JitsiConferenceEventManager.js

```diff
@@ -596,23 +596,17 @@ JitsiConferenceEventManager.prototype.removeXMPPListeners = function() {
 JitsiConferenceEventManager.prototype.setupXMPPListeners = function() {
     const conference = this.conference;
 
-    const featuresChangedListener = from => {
-        const participant
-            = conference.getParticipantById(
-            Strophe.getResourceFromJid(from));
+    const featuresChangedListener = (from, features) => {
+        const participant = conference.getParticipantById(Strophe.getResourceFromJid(from));
 
         if (participant) {
-            conference.eventEmitter.emit(
-                JitsiConferenceEvents.PARTCIPANT_FEATURES_CHANGED,
-                participant);
+            participant._features = features;
+            conference.eventEmitter.emit(JitsiConferenceEvents.PARTCIPANT_FEATURES_CHANGED, participant);
```

Comment:
maybe we could also add a parameter with the previous features? Otherwise calculating the difference is impossible since the previous ones are lost, right?

Reply:
This is fixed.

Reply:
This code has not changed from my PR. We can address it  later if needed.

Reply:
Remember the 11th commandment: Thou shalt leave the code better than it was. :-P